### ISSUE TITLE
warning: input out of order

### ISSUE THREAD

**Issue by dertin** [cracklib/cracklib#15]
```
Debian 8.8 x32 - Digitalocean 

# uname -a 
Linux framework 3.16.0-4-686-pae #1 SMP Debian 3.16.43-2+deb8u2 (2017-06-26) i686 GNU/Linux

```
```
# cd ../words
  
# make all
  
#  install -v -m644 -D  ./cracklib-words.gz /usr/share/dict/cracklib-words.gz
#  gunzip -v /usr/share/dict/cracklib-words.gz
#  ln -v -sf cracklib-words /usr/share/dict/words
#  install -v -m755 -d /usr/local/lib/cracklib

#  mkdir -p /usr/share/dict/cracklib-words
#  mkdir -p /usr/share/dict/cracklib-extra-words
#  create-cracklib-dict /usr/share/dict/cracklib-words /usr/share/dict/cracklib-extra-words
```

Console Error:

```
rm -f cracklib-words cracklib-words.gz cracklib-words.bz2
rm -f cracklib-words
bzcat -f `find files -type f` | tr A-Z a-z | sort | uniq > cracklib-words
rm -f cracklib-words.gz
gzip -v9 -c cracklib-words > cracklib-words.gz
cracklib-words:	 71.1%
rm -f cracklib-words.bz2
bzip2 -v9 -c cracklib-words > cracklib-words.bz2
  cracklib-words:  2.756:1,  2.903 bits/byte, 63.71% saved, 19163351 in, 6953913 out.
‘./cracklib-words.gz’ -> ‘/usr/share/dict/cracklib-words.gz’
/usr/share/dict/cracklib-words.gz:	 71.1% -- replaced with /usr/share/dict/cracklib-words
‘/usr/share/dict/words’ -> ‘cracklib-words’
...
warning: input out of order: 'ghabcdefghabcdefghabcdefghabcd' should not follow 'habcdefghabcdefghabcdefghabcde' (line 55362)
warning: input out of order: 'fghabcdefghabcdefghabcdefghabc' should not follow 'ghabcdefghabcdefghabcdefghabcd' (line 55363)
warning: input out of order: 'efghabcdefghabcdefghabcdefghab' should not follow 'fghabcdefghabcdefghabcdefghabc' (line 55364)
warning: input out of order: 'fghabcdefghabcdefghabcdefghabc' should not follow 'ghabcdefghabcdefghabcdefghabcd' (line 55366)
warning: input out of order: 'abcdefghi' should not follow 'fghabcdefghabcdefghabcdefghabc' (line 55367)
1911513 1911512
```

**Comment by dermotbradley** [comment]
Addressed by PR #33

**Comment by nneul** [comment]
This was merged, but still needs to be in a release.
